test/runtests.jl

```diff
@@ -129,8 +130,7 @@ end
 end
 
 ## Example with offsets from Venables & Ripley (2002, p.189)
-anorexia = readtable(joinpath(glm_datadir, "anorexia.csv.gz"))
-anorexia[:Treat] = pool(anorexia[:Treat])
+anorexia = CSV.read(joinpath(glm_datadir, "anorexia.csv"))
```

Comment:
Maybe pass `categorical=true`, that way we're safe if the default changes in a future version (as it's been discussed).

Reply:
According to the documentation the default value of `categorical` is `true`.  The reason for the explicit call here is that the levels are integers.  I could use an explicit `types` argument in the call to `CSV.read` but I don't think that would gain much.

Reply:
OK, I hadn't realized that `categorical=true` just means that the result will be categorical if the proportion of unique values is below a certain threshold. I guess that's OK. (I think we would have to pass `CategoricalValue` as the type to be certain we get a categorical vector.)